app/controllers/api/v1/book_suggestions_controller.rb

```diff
@@ -0,0 +1,17 @@
+module Api
+  module V1
+    class BookSuggestionsController < ApiController
+      def create
+        book_suggestion = BookSuggestion.new(book_suggestions_create_params)
+        book_suggestion.save!
```

Comment:
Still not a fan of this as I marked in #21 but the controller is not technically required for the card, so I'll approve it.